src/config/settings.ts

```diff
@@ -22,6 +23,18 @@ export let settings: Settings = {
   queries: {
     staleTime: 2000 * 60, // 2 minutes
   },
+  /**
+   * RPC URLs for different blockchains.
+   * Ensure these URLs are valid and accessible.
+   */
+  rpcUrls: {
+    near: "https://nearrpc.aurora.dev" as RpcUrl,
+    ethereum: "https://cloudflare-eth.com" as RpcUrl,
+    base: "https://mainnet.base.org" as RpcUrl,
+    arbitrum: "https://arb1.arbitrum.io/rpc" as RpcUrl,
+    bitcoin: "https://mainnet.bitcoin.org" as RpcUrl,
+    solana: "https://api.mainnet-beta.solana.com" as RpcUrl,
```

Comment:
Let's reuse chain names from `src/types/base.ts:14` and not make up new names for chains, it's becoming overwhelming to memorize what's what.